packages/http/generated-defs/TypeSpec.Http.ts

```diff
@@ -7,6 +7,19 @@ import type {
   Type,
 } from "@typespec/compiler";
 
+export interface QueryOptions {
+  readonly name?: string;
+  readonly explode?: boolean;
+  readonly format?: "multi" | "csv" | "ssv" | "tsv" | "simple" | "form" | "pipes";
```

Comment:
nit: why not just using explode, style to express both path/query/header parameter options like OpenAPI?

Reply:
you mean why format? this is because this was previously there